FORMAT_ADIOS.md

```diff
@@ -0,0 +1,28 @@
+# Implementation Details: ADIOS1/2
+
+## Datatype Conventions
+
+### Boolean Types
+
+ADIOS1 (BP3) was traditionally a C library and as such as no notion for boolean types.
+[At the moment](https://github.com/ornladios/ADIOS2/issues/2606), also ADIOS2 (BP4) does not define boolean types.
+Although the openPMD-standard avoids using boolean types itself, developers might choose to implement them as attributes or dataset types.
+
+For compatibility, boolean types shall be stored as `adios_unsigned_byte`/`unsigned char`.
```

Comment:
ADIOS2 maps `unsigned char` to `uint8_t`. But I think that `adios_unsigned_byte` is for ADIOS1, right? Also see [here](https://github.com/ornladios/ADIOS2/blob/8dd217ec720f11cf4e93e80b2db2a6953fa7b2fe/source/adios2/common/ADIOSTypes.cpp#L183).

Reply:
`uint8_t` might just be the `bpls` output since the introduction of fixes size types... let's check again @franzpoeschel

Reply:
In writing: `DefineVariable<unsigned char>()` will result in a `uint8_t` variable.
In reading: `IO::VariableType()` reports `"uint8_t"` which we [convert back again](https://github.com/openPMD/openPMD-api/blob/7d3e925b87c8c238405bda20ee6e2a11f6bf5d4c/src/IO/ADIOS/ADIOS2Auxiliary.cpp#L100) to `unsigned char`.